Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile for zookeeper-ldap-roles RPM.
+
+PWD=$(shell pwd)
+
+BLD=$(PWD)/build
+RPMDIR=$(BLD)/rpms
+
+REPO_TAG=$(shell git describe --tags HEAD --long)
+REPO_TAGFMT=$(shell echo ${REPO_TAG} | sed 's/-g/-/')
+REPO_VER=$(shell echo ${REPO_TAGFMT} | cut -d '-' -f 1)
+REPO_REL=$(shell echo ${REPO_TAGFMT} | cut -d '-' -f 2,3 | sed 's/-/_/')
+
+clean:
+	rm -rf $(BLD)
+
+install: 
+	bash gradlew fatJar
```

Comment:
why are you building fat jar? I figure we rather build (lean?) jar and install all dependencies separately? cause fat jar will include zookeeper jar as well?

Reply:
Changed Gradle command to build a lean jar.